canonicalwebteam/discourse/parsers/parsers.py

```diff
@@ -11,209 +11,213 @@
 )
 
 
-def _get_section(soup, title_text):
+class BaseParser(object):
```

Comment:
Can we change the file name from `parsers.py` to `base_parser.py`?

Reply:
Done